internal/test/framework/framework.go

```diff
@@ -229,8 +216,7 @@ func (f *TestFramework) getClusterAddress(config *restclient.Config) error {
 		return fmt.Errorf("API server has invalid format: expected hostname to start with `api.`")
 	}
 
-	// Replace `api.` with empty string for the first occurrence.
-	f.ClusterAddress = strings.Replace(hostName, "api.", "", 1)
```

Comment:
Why is this no longer needed?

Reply:
because we are getting the hostname in the format that we intend to use. so there is no more `api.` at the start.